addon/components/docs-keyboard-shortcuts/component.js

```diff
@@ -55,6 +55,12 @@ export default Component.extend(EKMixin, {
     }
   }),
 
+  hideKeyboardShortcuts: on(keyUp('Escape'), function() {
```

Comment:
Will this only trigger for this component or would this interfere with other `esc` listeners?

Reply:
I'm not very familiar with ember-keyboard, but based on a read-through of their ["priority" docs](https://patience-tema-baron.github.io/ember-keyboard/#/priority), my understanding is that this function will be called any time <kbd>Esc</kbd> is pressed while a) this `{{docs-keyboard-shortcuts}}` component is rendered and b) this component's `keyboardActivated` prop is `true`.

The ember-cli-addon-docs docs suggest rendering `{{docs-keyboard-shortcuts}}` in the `application.hbs`, and its `keyboardActivated` prop is set to `true` in its `init`, so for anyone who opts in to keyboard shortcuts it seems like yes, this callback function will be called on any docs page when <kbd>Esc</kbd> is pressed.

This is the case with all the other shortcuts already, though, and (again based on my reading of ember-keyboard docs) the keyup event's propagation won't be stopped, so it seems unlikely that it would interfere with any other keyup listeners on the page.

Reply:
@bantic gotcha, this seems fine to me then